src/components/SecondaryPanes/Breakpoints/Breakpoints.css

```diff
@@ -183,22 +183,12 @@ html .breakpoints-list .breakpoint.paused {
   inset-inline-end: 15px;
   inset-inline-start: auto;
   position: absolute;
-  top: 8px;
-  display: none;
+  top: -100px;
 }
 
 .breakpoint:hover .close-btn,
-.A11y-keyboard .breakpoint:focus-within .close-btn {
-  display: flex;
-}
-
-.breakpoint .close {
-  visibility: hidden;
-}
-
-.breakpoint:hover .close,
-.A11y-keyboard .breakpoint:focus-within .close {
-  visibility: visible;
+.close-btn:focus {
+  top: 25%;
```

Comment:
This only gives us correct vertical centering if the `.breakpoint` container is twice as high as the button.
I believe that button is 16px tall, so we are expecting a 32px tall row. I believe rows are currently a bit smaller than that, and should be changed to be 24px tall in an upcoming update.

Knowing the height of the button is 16px, a better value would be:
`top: calc(50% - 8px);`